autonode_service.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+set -e
+trap 'killall harmony' EXIT
+
+if ! pip3 list --format=legacy | grep AutoNode; then
+  echo "[AutoNode] Missing AutoNode python3 library, installing using pip3."
+  pip3 install AutoNode
+fi
+
+reset_mode=false
+
+node_dir=$(python3 -c "from AutoNode import common; print(common.node_dir)")
```

Comment:
shell strings with interporation must always use `""`, change the inner python to use `''`